### DISCUSSION TITLE
External GPS connection to LNM

### DISCUSSION THREAD

**Discussion by mlatapi** [albar965/littlenavmap#833] [Ideas]
Hi Alex,
I would like to know how if it si possible to connect a external GPS like Garmin to LNM
This will be very helpfull for pilots like me to have LNM updated during flight

**Comment by albar965** [comment]
Sorry, LNM does not support interfaces to GPS.

**Comment by danfly09** [comment]
I wonder if one can read the GPS coordinates from the Windows Surface PRO, that tablet runs Windows and has a built-in GPS. 
PS: https://stackoverflow.com/questions/36726068/accessing-location-on-windows-10-in-net-desktop-application
